example-projects/advanced/phantom/repl.js

```diff
@@ -1,10 +1,20 @@
-if (phantom.args.length != 1) {
-    console.log('Expected a target URL parameter.');
-    phantom.exit(1);
+var url;
+if (phantom.version.major > 1) {
+    var args = require('system').args;
+    if (args.length < 2) {
+        console.log('Expected a target URL parameter.');
```

Comment:
It seems that phantomjs doesn't write `console.error` to stderr though: https://github.com/ariya/phantomjs/issues/10150. Would need to do something like this to write to stderr: https://github.com/ariya/phantomjs/blob/master/examples/stdin-stdout-stderr.js